README.md

```diff
@@ -158,6 +158,6 @@ If you like this project please show support by staring ⭐️.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](/LICENSE) licensed.
+This project is [MIT](LICENSE.txt) licensed.
```

Comment:
- [ Optional ]  Kindly fix your MIT file link on the README.
![image](https://github.com/Obimbo07/tdn/assets/93598189/3f192c38-1907-421b-9a71-5fb35047d34a)